src/src/com/microsoft/aad/adal/TokenCacheItem.java

```diff
@@ -66,39 +66,76 @@
     private String mFamilyClientId;
 
     /**
-     * Construct default cache item.
+     * Default constructor for cache item.
      */
-    public TokenCacheItem() {
-
+    public TokenCacheItem() {}
+    
+    TokenCacheItem(final TokenCacheItem tokenCacheItem) {
+        this.mAuthority = tokenCacheItem.getAuthority();
```

Comment:
why did you add this. prefix everytwhere.... you already have prefis m
Choose only 1